_rules/inline-link-in-paragraph-is-distinguishable-be4d0c.md

```diff
@@ -0,0 +1,535 @@
+		border-color: red;
+		border-width: 0px;
+		border-bottom-width: 1px;
+	}
+</style>
+<p>Read about WAI on the <a class="test" href="http://w3.org/WAI">WAI webpage</a>.</p>
+```
+
+#### Passed Example 5
+
+This link, that is distinguishable by color (hue) from the other text in the same line, has a distinguishable box-shadow.
+
+```html
+<style>
+	p {
+		color: black;
+		background-color: white;
+	}
+	a.test {
+		color: blue;
+		background-color: white;
+		text-decoration: none;
+		box-shadow: 4px 4px;
+	}
+</style>
+<p>Read about WAI on the <a class="test" href="http://w3.org/WAI">WAI webpage</a>.</p>
+```
+
+#### Passed Example 6
+
+This link, that is distinguishable by color (hue) from the other text in the same line, has a distinguishable font-style.
+
+```html
+<style>
+	p {
+		color: black;
+		background-color: white;
+	}
+	a {
+		color: blue;
+		background-color: white;
+		text-decoration: none;
+		font-style: italic;
+	}
+</style>
+<p>Read about WAI on the <a href="http://w3.org/WAI">WAI webpage</a>.</p>
+```
+
+#### Passed Example 7
+
+This link, that is distinguishable by color (hue) from the other text in the same line, has an icon that makes it distinguishable as a link.
+
+```html
+<style>
+	p {
+		color: black;
+		background-color: white;
+	}
+	a.test {
+		color: blue;
+		background-color: white;
+		text-decoration: none;
+	}
+</style>
+<p>
+	Read about WAI on the
+	<a class="test" href="http://w3.org/WAI">WAI webpage <img src="/test-assets/be4d0c/icon.png" alt="" /></a>.
+</p>
+```
+
+#### Passed Example 8
+
+This link, that is distinguishable by color (hue) from the other text in the same line, has text that makes it distinguishable as a link.
+
+```html
+<style>
+	p {
+		color: black;
+		background-color: white;
+	}
+	a.test {
+		color: blue;
+		background-color: white;
+		text-decoration: none;
+	}
+</style>
+<p>To read about WAI, <a class="test" href="http://w3.org/WAI">follow this link to the WAI webpage</a>.</p>
+```
+
+#### Passed Example 9
+
+This link, that is distinguishable by color (hue) from the other text in the same line, uses a style similar to the default styling of links (underline) when it is focused or hovered.
+
+```html
+<style>
+	p {
+		color: black;
+		background-color: white;
+	}
+	a {
+		text-decoration: none;
+		color: blue;
+		background-color: white;
+	}
+	a:hover,
+	a:focus {
+		text-decoration: underline;
+	}
+</style>
+<p>Read about WAI on the <a href="http://w3.org/WAI">WAI webpage</a>.</p>
+```
+
+#### Passed Example 10
+
+This link, that is distinguishable by the background color (hue) from the other text in the same line, uses a style similar to the default styling of links (underline) when it is focused or hovered.
+
+```html
+<style>
+	p {
+		color: black;
+		background-color: white;
+	}
+	a {
+		color: black;
+		background-color: #c84d32;
+		text-decoration: underline;
+	}
+</style>
+<p>Read about WAI on the <a href="http://w3.org/WAI">WAI webpage</a>.</p>
+```
+
+#### Passed Example 11
```

Comment:
I feel there are too many examples now that the rules are merged…

This example, typically, doesn't show anything that Passed Examples 3 (role inherit from link) or 4 (distinguishing border, not underline) already show…

There are many moving pieces in that rule and I am not sure we need examples for all combinations. I'd rather go for a default case (Passed Example 1, essentially) + variation on a single theme (e.g. PE2: different bg is OK, PE3: role inheriting from link is OK; …) but no need to make all combinations imho.

Reply:
Reduced the number of passed examples, by removing some combinations